README.md

```diff
@@ -3,20 +3,22 @@
 This is a code extension designed so that we can code together during an on-site interview. Our guiding principal is: **we want to see you at your best** .
 
 ## Instructions
-If you bring your own computer to the interview, please ensure you are ready to go by having anything installed beforehand. If you want to use one of Reverb's computers, please tell us which code editor you prefer and which language you will be doing the extension in.
 
-## Details
+1. Choose which example (ruby or React) you would like to extend. If you don't feel comfortable in either of those 2 examples, feel free to implement your own (see "Implementing Your Own Solution"). Again, _we want to see the best side of you_ so please make sure you are comfortable with what we will be working on.
```

Comment:
I'll make sure to get my go one up and going by next week as well

Reply:
ya, no worries. No one has specifically requested Go yet.